### DISCUSSION TITLE
clang does not recognize qt-related header files

### DISCUSSION THREAD

**Discussion by 1925381584** [helix-editor/helix#11363] [General]
Hello, everyone. I have compile success a qt program in windows. but it could not find header file about qt. Whose problem is this and is there a solution for it?

system: win10
hx: 24.7

**Comment by pascalkuthe** [comment]
you need to setup a compilation database. Helix doesn't control the LSP and has no concept of a header file, libraries or compilation. This is most likely a config issue on your side (in theory it could be a bug with clangd but very unlikely, definitely not related to helix)

**Comment by 1925381584** [comment]
thanks for your replay. I'll look into it myself.

**Comment by jobukkit** [comment]
You just need to run `cmake -DCMAKE_EXPORT_COMPILE_COMMANDS=1 .` so that it produces the `compile_commands.json` file that `clangd` needs to understand the code.
